Sources/StreamChatUI/CommonViews/QuotedChatMessageView/QuotedChatMessageView.swift

```diff
@@ -238,6 +244,26 @@ open class QuotedChatMessageView: _View, ThemeProvider, SwiftUIRepresentable {
             preferredSize: attachmentPreviewSize
         )
     }
+    
+    /// Set the image from the given URL into `attachmentPreviewImage.image`
+    /// - Parameter url: The URL from which to generate the image on the video
+    open func setVideoAttachmentPreviewImage(url: URL?) {
+        guard let url = url else { return }
+        
+        DispatchQueue.global().async {
+            // Using AVKit to generate a preview video image
+            let asset = AVURLAsset(url: url)
+            let generator = AVAssetImageGenerator(asset: asset)
+            generator.appliesPreferredTrackTransform = true
+            
+            // Create a CGImage from the generator
+            if let cgImage = try? generator.copyCGImage(at: CMTime(seconds: 2, preferredTimescale: 60), actualTime: nil) {
+                DispatchQueue.main.async { [weak self] in
+                    self?.attachmentPreviewView.image = UIImage(cgImage: cgImage)
+                }
+            }
+        }
```

Comment:
We already have a function in `VideoLoading` component that does exactly this, we should use the `components.videoLoader` to generate the Video Image here too 👍

Reply:
Great spot @nuno-vieira pushing changes now 👍🏼